pep_sphinx_extensions/pep_theme/static/style.css

```diff
@@ -61,11 +61,9 @@ img.invert-in-dark-mode {
 :root {color-scheme: light dark}
 html {
     overflow-y: scroll;
-    margin: 0;
-    line-height: 1.4;
-    font-weight: normal;
+    line-height: 1.5;
     font-size: 1rem;
-    font-family: "Source Sans Pro", Arial, sans-serif;
+    font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica, Cantarell, Ubuntu, roboto, noto, arial, sans-serif;
```

Comment:
Quoting is not needed anymore for font names with spaces?

Reply:
Apparently not: https://github.com/tmcw/systemfontstack/pull/18